### PR TITLE
Add export tests

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Add tests for CSE export-corner processes and adapt old tests.

### CODE REVIEW

File: cypress/integration/run-processes/run-import-d2cc-process-acceptance-test.js [hunk 1]
```diff
@@ -30,31 +31,29 @@ Cypress.on('uncaught:exception', (err, runnable) => {
 
 describe('Run d2cc process', () => {
 
-   it('Run a full d2cc process ', () => {
+    it('Run a full d2cc process ', () => {
```
Comment (kahyami) [inline cypress/integration/run-processes/run-import-d2cc-process-acceptance-test.js:34]: It will be nice to mention that this is the import process

Comment (TheMaskedTurtle) [reply cypress/integration/run-processes/run-import-d2cc-process-acceptance-test.js]: renamed

**Review by kahyami — APPROVED** [review]
